config/admins.txt

```diff
@@ -12,3 +12,4 @@ duncandelano = Game Master
 ecolli = Game Master
 crazypilot = Game Master
 vivalas = Coder
+Qustinnus = Game Master
```

Comment:
its not even a meme im the headmin anyway and I hate having to change this locally every time.